### PR TITLE
Enhancement/Azure New Rules

### PR DESCRIPTION
**Pull request by JJmako**
Added two new rules.

### CODE REVIEW

**Comment by j4v** [comment]
Please always make PRs against `develop` (see https://github.com/nccgroup/ScoutSuite/blob/master/CONTRIBUTING.md).

File: ScoutSuite/providers/azure/rules/findings/storageaccount-default-network-access.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "dashboard_name": "Storage Accounts",
+    "description": "Default Network Access Allowed",
+	"rationale": "<b>Description:</b><br><br>Restricting default network access helps to provide a new layer of security, since storage accounts accept connections from clients on any network. To limit access to selected networks, the default action must be changed..<br><br><b>References:</b><ul><li>CIS Microsoft Azure Foundations v1.0.0 3.3</li></ul>",
```
Comment (j4v) [inline ScoutSuite/providers/azure/rules/findings/storageaccount-default-network-access.json]: The CIS reference looks wrong, should be `v1.1.0 3.7`.

File: ScoutSuite/providers/azure/rules/findings/storageaccount-trusted-microsoft-services.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "dashboard_name": "Storage Accounts",
+    "description": "Trusted Microsoft Services enabled",
+	"rationale": "<b>Description:</b><br><br><br><br><b>References:</b><ul><li>CIS Microsoft Azure Foundations v1.0.0 3.3</li></ul>",
```
Comment (j4v) [inline ScoutSuite/providers/azure/rules/findings/storageaccount-trusted-microsoft-services.json]: The CIS reference looks wrong, should be `v1.1.0 3.8`.

File: ScoutSuite/providers/azure/rules/findings/storageaccount-default-network-access.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (j4v) [inline ScoutSuite/providers/azure/rules/findings/storageaccount-default-network-access.json]: What's the difference between this one and `ScoutSuite/providers/azure/rules/findings/storageaccount-public-blob-container.json`?

Comment (JJmako) [reply ScoutSuite/providers/azure/rules/findings/storageaccount-default-network-access.json]: If I am not wrong, the ScoutSuite/providers/azure/rules/findings/storageaccount-public-blob-container.json refers to 3.6 point (Ensure that 'Public access level' is set to Private for blob containers) and this refers to 3.7 (Ensure default network access rule for Storage Accounts is set to deny)

**Comment by j4v** [comment]
@JJmako this is approved, feel free to merge!

**Pull request merged**